common/presenters/events-to-timeline-component.js

```diff
@@ -0,0 +1,66 @@
+const i18n = require('../../config/i18n')
+const addTriggeredEvents = require('../helpers/events/add-triggered-events')
+const setEventDetails = require('../helpers/events/set-event-details')
+
+const getItem = ({
+  component,
+  labelClasses,
+  heading,
+  description,
+  timestamp,
+  byline,
+}) => {
+  return {
+    component,
+    label: {
+      classes: labelClasses,
+      html: heading,
+    },
+    classes: 'app-timeline__item',
+    html: description,
+    datetime: {
+      timestamp,
+      type: 'datetime',
+    },
+    byline: {
+      html: byline,
+    },
+  }
+}
+
+const eventsToTimelineComponent = (moveEvents, move) => {
+  moveEvents = addTriggeredEvents(moveEvents)
```

Comment:
Rather than passing the whole thing in, could these not use `.map()`?

It would make testing the helpers simpler too because they just have simple input/output too.

Reply:
No, we have to pass the whole thing into `addTriggeredEvents` because it inserts the additional faux triggered events.

Reply:
Okay. Can we assign this to a new variable so that we don't risk mutating the argument.

Reply:
I don't see how this risks mutating the argument. Anyhow, I've passed the addTriggeredEvents directly to the map, removing the need for the reassignment.